beets/ui/__init__.py

```diff
@@ -1057,43 +1048,149 @@ def _open_library(config):
     return lib
 
 
-def _raw_main(args, lib=None):
+class BeetsCLI(click.MultiCommand):
+
+    def _commands(self, ctx):
+        from beets.ui.commands import default_commands
```

Comment:
Wondering if we make `default_commands` a dict and therefore remove some work from `_commands`. This also helps with the command aliases such as `ls`.